docs/config.md

```diff
@@ -212,6 +212,20 @@ Starting with `branding`, the following subproperties are available:
 3. `auth_footer_links`: A list of links to add to the footer during login, registration, etc. Each entry must have a `text` and
    `url` property.
 
+
+4. `title_template`: A template string that can be used to configure the title of the application when not viewing a room.
+5. `title_template_in_room`: A template string that can be used to configure the title of the application when viewing a room
+
+#### `title_template` vars
```

Comment:
There is an issue here that this (by omission) assures these values to be stable. Should we add a disclaimer that these parameters may change at any time?

Reply:
Cool. I'll put this under product review then. My understanding is the feature itself was greenlit, though I think we should be aware of the configuration implications.